MPF.md

```diff
@@ -1479,31 +1473,32 @@ be defined in an [augments file][Augments].
 
 For example:
 
-```
+```json
 {
-  "classes" {
+  "classes": {
     "client_initiated_reporting_enabled": [ "any" ]
   },
   "vars": {
-    "control_server_call_collect_interval": "1",
+    "control_server_call_collect_interval": "1"
   }
 }
 ```
 
 ### Configure client initiated reporting timeout
 
-By default cf-serverd holds an open connection for client initiated for 30
+By default `cf-serverd` holds an open connection for client initiated for 30
 seconds. In some environments this value may need to be increased in order for
 report collection to finish. Once the connection has been open for longer than
 the specified seconds it is closed.
 
 The window of time can be controled by setting `def.control_server_collect_window`.
 
-For example, enable client initated reporting for all hosts with a 10 minute interval and hold the connection open for 90 seconds.
+For example, enable client initiated reporting for all hosts with a 10 minute
```

Comment:
why change one line (nice) to two lines here? Doesn't help result right?

Reply:
though I do see just below and above we are 80-column formatting in general :( so I guess, OK. 😢

Reply:
Just trying to increase consistency, i.e. be consistent with what's around.